src/simularium/localSimulators/IClientSimulator.ts

```diff
@@ -0,0 +1,36 @@
+import { TrajectoryFileInfoV2, VisDataMessage } from "../types";
+
+export interface IClientSimulator {
+    update(dt: number): VisDataMessage;
+    getInfo(): TrajectoryFileInfoV2;
+}
+
+// TODO these should not be needed anywhere except ClientSimulatorConnection
+export const enum ClientMessageEnum {
+    ID_UNDEFINED_WEB_REQUEST = 0,
+    ID_VIS_DATA_ARRIVE = 1,
+    ID_VIS_DATA_REQUEST = 2,
+    ID_VIS_DATA_FINISH = 3,
+    ID_VIS_DATA_PAUSE = 4,
+    ID_VIS_DATA_RESUME = 5,
+    ID_VIS_DATA_ABORT = 6,
+    ID_UPDATE_TIME_STEP = 7,
+    ID_UPDATE_RATE_PARAM = 8,
+    ID_MODEL_DEFINITION = 9,
+    ID_HEARTBEAT_PING = 10,
+    ID_HEARTBEAT_PONG = 11,
+    ID_TRAJECTORY_FILE_INFO = 12,
+    ID_GOTO_SIMULATION_TIME = 13,
+    ID_INIT_TRAJECTORY_FILE = 14,
+    // insert new values here before LENGTH
+    LENGTH,
+}
+
+// TODO these should not be needed anywhere except ClientSimulatorConnection
+export const enum ClientPlayBackType {
+    ID_LIVE_SIMULATION = 0,
+    ID_PRE_RUN_SIMULATION = 1,
+    ID_TRAJECTORY_FILE_PLAYBACK = 2,
+    // insert new values here before LENGTH
+    LENGTH,
+}
```

Comment:
Is it possible to avoid having these enums in 2 places (in this file and in NetConnection.ts)?

Reply:
the only one that matters is the one in netconnection.  the ones in here are totally artificial.  I'll think about removing them but they don't affect the ISimulator / SimulariumController operations.